2020-04-17 02:39:46,454 ERROR: Exception on /RavenScout [GET] [in c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\routes.py", line 178, in RavenScout
    return render_template('RavenScout.html', title='About')
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\RavenScout.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 3, in block "doc"
    <html{% block html_attribs %}{% endblock html_attribs %}>
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 11, in block "html"
    {%- endblock metas %}
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 13, in block "body"
    {%- block styles %}
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\base.html", line 52, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\RavenScout.html", line 55, in block "app_content"
    <br><br><img src="{{ url_for('img', filename='diagram.jpg') }}">
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'img' with values ['filename']. Did you mean 'home' instead?
2020-04-17 02:40:32,452 INFO: Microblog startup [in C:\Users\Daniel\Documents\GitHub\RavenScout\app\__init__.py:51]
2020-04-17 02:40:35,297 ERROR: Exception on /RavenScout [GET] [in c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\routes.py", line 178, in RavenScout
    return render_template('RavenScout.html', title='About')
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\RavenScout.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 3, in block "doc"
    <html{% block html_attribs %}{% endblock html_attribs %}>
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 11, in block "html"
    {%- endblock metas %}
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 13, in block "body"
    {%- block styles %}
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\base.html", line 52, in block "content"
    {% block app_content %}{% endblock %}
  File "C:\Users\Daniel\Documents\GitHub\RavenScout\app\templates\RavenScout.html", line 55, in block "app_content"
    <br><br><img src="{{ url_for('img', filename='diagram.jpg') }}"/>
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\app.py", line 2215, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\flask\helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "c:\users\daniel\appdata\local\programs\python\python38-32\lib\site-packages\werkzeug\routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'img' with values ['filename']. Did you mean 'home' instead?
2020-04-17 02:41:27,823 INFO: Microblog startup [in C:\Users\Daniel\Documents\GitHub\RavenScout\app\__init__.py:51]
2020-04-17 02:42:01,202 INFO: Microblog startup [in C:\Users\Daniel\Documents\GitHub\RavenScout\app\__init__.py:51]
2020-04-17 02:46:12,507 INFO: Microblog startup [in C:\Users\Daniel\Documents\GitHub\RavenScout\app\__init__.py:51]
